app/views/admin/organizations/_form.html.haml

```diff
@@ -0,0 +1,13 @@
+= semantic_form_for(@organization, url: (@organization.new_record? ? admin_organizations_path : admin_organization_path(@organization))) do |f|
+  = f.inputs name: 'Organization details' do
+    = f.input :name, as: :string, required: true
+    = f.input :description, as: :text, input_html: { rows: 10 }, placeholder: 'Decribe about your organization..'
```

Comment:
https://github.com/justinfrench/formtastic#the-available-inputs It says `:text` is used for a textarea in formtastic.